core/state_processor_test.go

```diff
@@ -498,8 +498,49 @@ func TestProcessVerkle(t *testing.T) {
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* write code hash */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #0 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #1 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #2 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #3 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #4 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #5 */
+			params.WitnessChunkReadCost+ /* SLOAD in constructor */
+			params.WitnessChunkWriteCost+ /* SSTORE in constructor */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+params.WitnessBranchReadCost+params.WitnessBranchWriteCost+ /* creation (CREATE at PC=0x121) */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* write code hash */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #0 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #1 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #2 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #3 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #4 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #5 */
+			params.WitnessChunkReadCost+ /* SLOAD in constructor */
+			params.WitnessChunkWriteCost+ /* SSTORE in constructor */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* write code hash for tx creation */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #0 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #1 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #2 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #3 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #4 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #5 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #6 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #7 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #8 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #9 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #10 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #11 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #12 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #13 */
+			params.WitnessChunkReadCost+params.WitnessChunkWriteCost+ /* code chunk #14 */
+			4844 /* execution costs */)
```

Comment:
I had always "aced" the gas costs. I think it's a better idea to break it down one by one.

I spent way too much time on this, and the execution costs - 4844 - is suspicious. But this test passes. The others don't at the moment.